File: 1006/1006.cs
using System;
class URI {
    static void Main(string[] args) {
        double a, b, c, r;

        a = double.Parse(Console.ReadLine());
        b = double.Parse(Console.ReadLine());
        c = double.Parse(Console.ReadLine());
        r = (a * 2 + b * 3 + c * 5) / 10;

        Console.WriteLine("MEDIA = " + r.ToString("F1"));
    }
}

File: 1006/1006.swift
import Foundation

if let inputA = readLine(), let inputB = readLine(), let inputC = readLine(), let a = Double(inputA), let b = Double(inputB), let c = Double(inputC) {
    let r = (a * 2 + b * 3 + c * 5) / 10
    print(String(format: "MEDIA = %.1f",  r))
}